sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 4.17.0-beta.1 (Unreleased)
+
+### Features Added
+New Event Grid client for Namespace feature.
```

Comment:
Is there a link to learn.microsoft.com that you can add here or a link to the README section that describes what this feature is?